src/tests/common.inc

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use AWS\CRT\CRT as CRT;
+
+abstract class CrtTestCase extends PHPUnit_Framework_TestCase {
+    // Ensure that after every test there are no errors in the CRT itself
+    protected function assertPostConditions() {
+        $this->assertEquals(0, CRT::last_error(), "Test left an error on the stack: " . CRT::error_name(CRT::last_error()));
```

Comment:
More itiomatic way of doing it would be 
```
if( CRT::last_error()) {
      self::fail("Test left an error on the stack: " . CRT::error_name(CRT::last_error())));
}
```
(syntax may be wrong)